app/test_main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+from unittest import mock
+from typing import Callable
+from app.main import can_access_google_page
+
+
+def test_can_access_google_page() -> None:
+    assert can_access_google_page("https://mate.academy/learn/"
+                                  "python-core/python-core-"
+                                  "testing-in-details?"
+                                  "learnItemsFilter="
+                                  "All&section=video"
+                                  "&theoryId=3664&videoId"
+                                  "=2193&testTaskSlug="
+                                  "py_can_access_google_page") == "Accessible"
```

Comment:
The test name `test_can_access_google_page` could be more descriptive. According to the checklist, test names should be descriptive and correct. Consider renaming it to something like `test_accessible_url_with_valid_conditions` to better reflect what the test is verifying.